content/applications/productivity/discuss/chatter.rst

```diff
@@ -0,0 +1,407 @@
+updates a follower is subscribed to, and to edit the list, click the :guilabel:`👤 (user)` icon.
+Find the appropriate follower in the list, then click the :guilabel:`✏️ (pencil)` icon. This opens
+the :guilabel:`Edit Subscription` pop-up window for the follower.
+
+The list of available subscription settings varies depending on the record type. For example, a
+follower of a *Helpdesk* ticket may be informed when the ticket is rated. This option would not be
+available for the followers of a *CRM* opportunity.
+
+Check the checkbox for any updates the follower should receive, and clear the checkbox for any
+updates they should **not** receive. Click :guilabel:`Apply` when finished.
+
+.. figure:: chatter/chatter-edit-subscription.png
+   :align: center
+   :alt: The Edit Subscription window on a Helpdesk ticket.
+
+   The Edit Subscription options vary depending on the record type. These are the options for a
+   Helpdesk ticket.
+
+.. _discuss/log-notes:
+
+Log notes
+=========
+
+The chatter function includes the ability to log internal notes on individual records. These notes
+are only accessible to internal users, and are available on any records that feature a chatter
+thread.
+
+To log an internal note, first navigate to a record. For example, to open a *CRM* opportunity,
+navigate to :menuselection:`CRM app --> Sales --> My Pipeline` and click on the Kanban card of an
+opportunity to open it. Then, at the top-right above the chatter composer, click :guilabel:`Log
+note`.
+
+Enter the note in the chatter composer. To tag an internal user, type `@`, then select a name from
+the drop-down menu. Depending on their notification settings, the user is notified by email or
+through Odoo.
+
+.. tip::
+   Outside contacts can also be tagged in an internal log note. The contact then receives an email
+   with the contents of the note they were tagged in, including any attachments added directly to
+   the note. If they respond to the email, their response is logged in the chatter, and they are
+   added to the record as a follower.
+
+   Outside contacts are **not** able to log in to view the entire chatter thread, and are only
+   notified of specific updates, based on their :ref:`follower subscription settings
+   <discuss/edit-subscription>`.
+
+.. _discuss/send-messages:
+
+Send messages
+=============
+
+Chatter *composers* can send messages to outside contacts, without having to leave the database, or
+open a different application. This makes it easy to communicate with potential customers in the
+*Sales* and *CRM* applications, or vendors in the *Purchase* app.
+
+To send a message, first navigate to a record. For example, to send a message from a *CRM*
+opportunity, navigate to :menuselection:`CRM app --> Sales --> My Pipeline` and click on the Kanban
+card of an opportunity to open it. Then, at the top-right above the chatter composer, click
+:guilabel:`Send message`.
+
```

Comment:
example blocks for the other app references you have? just a thought-- could go either way